components/toast_wrapper/index.ts

```diff
@@ -44,8 +52,8 @@ export function makeCountUnreadsBelow() {
         'makeCountUnreadsBelow',
         getAllPosts,
         getCurrentUserId,
-        (state, postIds) => postIds,
-        (state, postIds, lastViewedBottom) => lastViewedBottom,
+        (state: GlobalState, postIds?: string[] | null) => postIds,
+        (state: GlobalState, postIds: string[] | null | undefined, lastViewedBottom: number) => lastViewedBottom,
```

Comment:
We don't need to pass undefined or null, we can declare is as an empty array when the id's aren't present

Reply:
Good point, thanks